src/setuptools_scm/version.py

```diff
@@ -141,7 +141,7 @@ def __init__(
             distance = 0
         self.distance = distance
         self.node = node
-        self.time = datetime.datetime.now()
+        self.time = datetime.datetime.now(utc)
```

Comment:
im wondering if a switch to `utcnow` would make sense for the older pythons

Reply:
My problem with tat is, it  creates a datetime object with utc time, but does not attach timezone information to the object.

Reply:
If you really really want me to use `utcnow`, please give me another ping, and I'll change the PR accordingly. But I recommend against it.

Reply:
It's fine as is, ill coin this as a major release, im still adjusting to daycare shutdown and the house arrest from corona, since I have a toddler i chances for Open-source are rare